src/components/test/Spinner-test.js

```diff
@@ -9,25 +9,73 @@ describe('Spinner', () => {
 
   it('renders', () => {
     const wrapper = createSpinner();
-    assert.isTrue(wrapper.find('SvgIcon.Hyp-Spinner--medium').exists());
+    assert.isTrue(wrapper.find('Icon .Hyp-Spinner--medium').exists());
   });
 
   it('uses the registered `hyp-spinner` icon', () => {
     const wrapper = createSpinner();
     assert.equal(
-      wrapper.find('SvgIcon').props().name.toString(),
+      wrapper.find('Icon').props().name.toString(),
       Symbol('spinner').toString()
     );
   });
 
   it('applies additional classes', () => {
     const wrapper = createSpinner({ classes: 'foo bar' });
-    assert.isTrue(wrapper.find('SvgIcon.Hyp-Spinner--medium.foo.bar').exists());
+    assert.isTrue(wrapper.find('Icon .Hyp-Spinner--medium.foo.bar').exists());
   });
 
   it('sets indicated size', () => {
     const wrapper = createSpinner({ size: 'large' });
-    assert.isTrue(wrapper.find('SvgIcon.Hyp-Spinner--large').exists());
+    assert.isTrue(wrapper.find('Icon .Hyp-Spinner--large').exists());
+  });
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility({
+      content: () => createSpinner(),
+    })
+  );
+});
+
+describe('FullScreenSpinner', () => {
+  const createSpinner = (props = {}) => mount(<FullScreenSpinner {...props} />);
+
+  it('renders a containing div', () => {
+    const wrapper = createSpinner();
+    assert.isTrue(wrapper.find('div.Hyp-FullScreenSpinner').exists());
+  });
+
+  it('renders a spinner', () => {
+    const wrapper = createSpinner();
+
+    const spinner = wrapper.find('Spinner');
+    assert.isTrue(spinner.exists());
+    assert.equal(spinner.prop('size'), 'large');
+  });
+
+  it('applies additional classes', () => {
+    const wrapper = createSpinner({ classes: 'foo bar' });
+    const spinner = wrapper.find('Spinner');
+
+    assert.equal(
+      spinner.prop('classes'),
+      'Hyp-FullScreenSpinner__spinner foo bar'
+    );
+  });
+
+  it('applies container classes', () => {
+    const wrapper = createSpinner({ containerClasses: 'foo bar' });
+
+    assert.deepEqual(
+      [
+        ...wrapper
+          .find(`div.Hyp-FullScreenSpinner.foo.bar`)
```

Comment:
The `find` selector seems overly specific here, since you're already checking the exact classes on the matching element.